bin/node/runtime/src/lib.rs

```diff
@@ -600,7 +600,7 @@ impl pallet_staking::Config for Runtime {
 	type TargetList = pallet_staking::UseValidatorsMap<Self>;
 	type MaxUnlockingChunks = ConstU32<32>;
 	type HistoryDepth = HistoryDepth;
-	type OnStakerSlash = NominationPools;
+	type EventListeners = NominationPools;
```

Comment:
In future, will this need to be a vec/tuple? Since both stake tracker and nomination pool needs to listen to staking events?

Reply:
Never mind, realised its implemented for tuples.